### PR TITLE
Add describe_process #269

### PR DESCRIPTION
**Pull request by m-mohr**
Implements #269

### CODE REVIEW

**Comment by clausmichele** [comment]
As soon as I have time I will test it! Thanks

File: openeo/rest/connection.py [hunk 1]
```diff
@@ -719,6 +719,24 @@ def list_processes(self, namespace: str = None) -> List[dict]:
             processes = self.get('/processes/' + namespace, expected_status=200).json()["processes"]
         return VisualList("processes", data=processes, parameters={'show-graph': True, 'provide-download': False})
 
+    def describe_process(self, id: str, namespace: str = None) -> dict:
+        """
+        Retuens a single process from the back end.
```
Comment (soxofaan) [inline openeo/rest/connection.py:724]: typo "Retuens"

File: openeo/rest/connection.py [hunk 2]
```diff
@@ -719,6 +719,24 @@ def list_processes(self, namespace: str = None) -> List[dict]:
...
+
+        :param id: The id of the process.
+        :param namespace: The namespace of the process.
+
+        :return: process: Dict The process definition.
```
Comment (soxofaan) [inline openeo/rest/connection.py:729]: I know you copied this from `list_processes` (were it is also wrong), but this `Dict` does not belong there, the doc generator tool gets that from the function return type hint. It should just be

> :return: The process definition.

**Review by soxofaan — COMMENTED** [review]
looks great (just some small doc issues)

File: openeo/rest/connection.py (hunk 2 above)
Comment (m-mohr) [reply openeo/rest/connection.py:729]:
```suggestion
        :return: The process definition.
```

File: openeo/rest/connection.py (hunk 1 above)
Comment (m-mohr) [reply openeo/rest/connection.py:724]:
```suggestion
        Returns a single process from the back end.
```

**Comment by m-mohr** [comment]
Thanks, @soxofaan. I've fixed the points you have raised.

**Pull request merged**